org.ekstep.lessonbrowser-1.4/editor/lessonBrowserApp.js

```diff
@@ -422,7 +422,8 @@ angular.module('org.ekstep.lessonbrowserapp', ['angular-inview', 'luegg.directiv
                         var contents = [];
                         $scope.mainTemplate = 'facetsItemView';
                         ctrl.applyAllJquery();
-                        if (ctrl.facetsResponse.result.response.sections) {
+                        var contentCount = _.findIndex(ctrl.facetsResponse.result.response.sections, function(o) { return o.contents });
```

Comment:
why you want to do `_.findIndex ` ? 



let say what if `o.contents = []` ?

```js 
var sections = [
  { 'contents': [] }
];

_.findIndex(users, function(o) { return o.user });

```
Will your code works in this above case? @gouravmore

Reply:
Yes, this code will not work in the above case,  but the Assemble API return `"contents": null` in the result. 
As per the above example the `_.findIndex` will return -1 if response as follows
```js 
var sections = [
  { 'contents': null }
];

_.findIndex(users, function(o) { return o.user });
=> -1
```

Reply:
so the previous code will work right??

Reply:
You should tell

Reply:
This will not work if we perform this operation on objects and apply check on variable
```js
var sections = {
  'contents': []
};

var count  = _.findIndex(sections, function(o) { return o.contents });
// -1
```
the follwoing code will not work```
```js
if(count){
 // true
}else{ 
// false
}

output=> true